### PR TITLE
New json parser

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: packages/basics/package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
   },
   "homepage": "https://github.com/Inist-CNRS/ezs/tree/master/packages/basics#readme",
   "dependencies": {
-    "JSONStream": "~1.3.5",
```
Comment (parmentf) [inline packages/basics/package.json:22]: Why remove it from dependencies ?
If default option is to use legacy parser...

File: packages/basics/test/json-parse.js [hunk 2]
```diff
@@ -15,10 +15,21 @@ describe('JSONParse', () => {
                 done();
             });
     });
+    it('should return a value (yajs)', (done) => {
+        from(['{ "a": 1 }'])
+            .pipe(ezs('JSONParse', { legacy: true }))
```
Comment (parmentf) [inline packages/basics/test/json-parse.js:20]: Should'nt it be `legacy: false`.
README says that default value is `true`.

File: packages/basics/package.json (hunk 1 above)
Comment (touv) [reply packages/basics/package.json:22]: you're right thanks

File: packages/basics/test/json-parse.js (hunk 2 above)
Comment (touv) [reply packages/basics/test/json-parse.js:20]: right, too

**Pull request merged**